src/compiler/worker/runWebpackCompiler.js

```diff
@@ -50,6 +50,8 @@ module.exports = function runWebpackCompiler({
 
   let lastPercent = -1;
 
+  config.output.path = configOptions.assetsDest;
```

Comment:
I'd rather like to update the `getConfig` to handle `assetsDest` than mutating stuff like that